# Differential Gene Expression Analysis of Common Bean Lineages During Development

![cm2](https://user-images.githubusercontent.com/92789637/184629535-4f2bf195-b137-4b0f-92ef-b839330f89f3.jpg)


Common bean(Phaseolus vulgaris) is an annual leguminous plant grown majorly for its edible dry seeds. It is a protein-rich legume with vital proteinaceous nutrients, carbohydrates, some vitamins and micronutrients. 

Despite its nutritional importance, its adoption in diets is restricted by a phenomenon known as the hard-to-cook defect. The HTC defect is characterised by longer cooking time and greatly affects the acceptability and utility of the variety by the consumer market. Longer cooking time in addition lowers nutritive value. Understanding of the mechanisms leading to this phenomenon would provide insight into the development of the defect itself and would aid in the search for appropriate methods to prevent and reverse it.

Transcriptomic profiling in three common bean varieties, fast-cooking (cv. Rosecoco), a slow-cooking (cv. Pinto) and intermediate (cv. Canadian wonder) is explored in this study to determine differentially expressed genes that may contribute to HTC development.

## **Aim and Objectives**

The aim is to identify differentially expressed genes during seed development useful in the development of hard-to-cook defect. 

## **Workflow**
![workflow](https://user-images.githubusercontent.com/92789637/187473859-d4ddf7bf-37f1-4ec4-963e-4994275b094a.png)


## **Data Sources**

Differential gene expression analysis will be carried out using pubicly available RNA seq data of the three comon bean varieties from [NCBI-SRA](https://github.com/omicscodeathon/commonbean_degs/blob/main/accessions/acc_list52.txt)

## **Contributors**

1. Olaitan Awe

2. Brenda Kiage
